internal/services/sentinel/sentinel_watchlist_item_resource_test.go

```diff
@@ -0,0 +1,190 @@
+package sentinel_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/sentinel"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/sentinel/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type WatchlistItemResource struct {
+	Name string
+}
+
+func TestAccWatchlistItem_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_sentinel_watchlist_item", "test")
+	name, _ := uuid.GenerateUUID()
```

Comment:
that we're defining this in each test configuration would indicate that the resource should be generating those, and this field should be optional/computed (or even potentially removed altogether) instead?

Reply:
This is the name of the watchlist item, which is a part of the URL that we have to construct.